### DISCUSSION TITLE
Bagaimanakah mengganti logo icon font awesome di infobox shiny dg gambar menggunakan R Studio ?

### DISCUSSION THREAD

**Discussion by Micoalgo** [teamalgoritma/community#600] [Q&A] [answered]
setelah dicoba kode sbb ;
<img width="521" alt="image" src="https://user-images.githubusercontent.com/123436963/227769445-d32cc6db-38fb-4ec2-ba4c-b97b3fdb459e.png">

utk mengganti Icon font awesome di infobox shiny dg gambar, kemudian muncul error 
<img width="272" alt="image" src="https://user-images.githubusercontent.com/123436963/227769503-c47d5700-77b8-4bff-a3f9-27607b4dd587.png">

Bagaimanakah cara memperbaiki kesalahan pada kode tsb agar icon berubah menjadi image pada infobox tsb ? Terimakasih

**Comment by Yusraf002** [comment]
Dear @Micoalgo, 
Terima kasih atas pertanyaannya pak, tim kami akan segera coba bantu atau carikan referensi ya pak

**Comment by Micoalgo** [comment]
Terimakasih

**Comment by idoali** [comment, marked as answer]
Terima kasih pak atas pertanyaannya

Bapak bisa mencoba library `apputils` pak. Sebelum digunakan, tentunya jangan lupa library `apputils` di install terlebih dulu. Kurang lebih penggunaannya pada kasus bapak adalah seperti ini :

```{r}
apputils::infoBox(title = "1. OVERVIEW",
		  value = "Dashboard ini memberikan gambar",
		  icon = apputils::icon(list(src = "lgrsn.png", width = "80px"), lib = "local"),
		  width = 12,
		  color = "aqua")
```

Semoga cukup membantu pak.

**Reply by Micoalgo** [reply]
Terimakasih sebelumnya Mas Ido,
Saya coba cari packages apputils nya di project  R studio tsb tapi tdk ketemu, carinya dimana ya ?

**Reply by IrfanChairurrachman** [reply]
Saya izin bantu menjawab cara menginstall library appituls.

Bapak dapat membuka R Studio console kemudian menjalankan command berikut:
`devtools::install_github("leonawicz/apputils")`

Notes: pastikan telah menginstall package `devtools` 

Sekian
Terima kasih

**Reply by Micoalgo** [reply]
Terimakasih Mas Irfan
